generators/liquibase/support/prepare-field.js

```diff
@@ -139,14 +139,27 @@ function parseLiquibaseLoadColumnType(entity, field) {
 }
 
 export default function prepareField(entity, field) {
+  field.fieldDefaultValueDefined = field.defaultValue !== undefined;
+  field.fieldHasAnyDefaultValue = field.fieldDefaultValueDefined || field.defaultValueComputed !== undefined;
+
   mutateData(field, {
     __override__: false,
     columnType: data => parseLiquibaseColumnType(entity, data),
-    shouldDropDefaultValue: data => data.fieldType === ZONED_DATE_TIME || data.fieldType === INSTANT,
+    shouldDropDefaultValue: data => !data.fieldHasAnyDefaultValue && (data.fieldType === ZONED_DATE_TIME || data.fieldType === INSTANT),
```

Comment:
Move `liquibaseDefaultValueAttributeValue` and `liquibaseDefaultValueAttributeName` before `shouldDropDefaultValue`.
`liquibaseDefaultValueAttributeName` can be used instead.
```suggestion
    shouldDropDefaultValue: data => !data.liquibaseDefaultValueAttributeValue && (data.fieldType === ZONED_DATE_TIME || data.fieldType === INSTANT),
```